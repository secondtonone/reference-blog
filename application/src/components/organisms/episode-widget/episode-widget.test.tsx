/**
 * @jest-environment jsdom
 */
import { render } from '@testing-library/react';
import EpisodeWidget from '.';

describe('Match snapshots EbookWidget', () => {
  it('default', () => {
    const { asFragment } = render(
      <EpisodeWidget id="5768" />
    );

    expect(asFragment()).toMatchSnapshot();
  });
});
