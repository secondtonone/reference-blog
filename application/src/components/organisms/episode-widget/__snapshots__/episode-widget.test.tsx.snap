// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Match snapshots EbookWidget default 1`] = `
<DocumentFragment>
  <div
    class="box-adaptive__BoxStyled-sc-g2wy39-0 ksNfYJ"
  />
</DocumentFragment>
`;
